app/models/person.rb

```diff
@@ -1,17 +1,19 @@
 class Person < ActiveRecord::Base
   #code to get next un-used id # in the 800 range
   #((801...900).to_a - (array_from_db_of_taken_numbers))[0]
-  before_save :title_order
+  attr_accessible :firstname, :lastname, :gender, :date_of_birth, :zipcode, :city, :state, :status, :end_date, :department, :icsid, :title, :title_ids, :start_date, :division1, :division2, :comments
+  attr_accessible :channels_attributes
 
-  attr_accessible :firstname, :lastname, :status, :icsid, :department, :city, :state, :zipcode, :start_date, :end_date , :title, :gender, :date_of_birth,:division1, :division2, :channels_attributes, :title_ids, :title_order, :comments
```

Comment:
I have some general feedback, which I hope you find helpful, and then a question about the problem you solved with these two lines (4-5).

General feedback:

The difference between the line 6 that was removed, and the lines 4-5 that were added, is the removal of one `attr_accessible` field.  However, because you changed the order of the attributes, it took me a few minutes to figure out the difference.

I think it is considerate to make changes that minimize incidental impact.  Anything can be improved, and the project benefits!  But I'd suggest making changes that make things better and are related to the purpose of the PR.  Does changing the order of the fields have a benefit?

If your change shortened these lines so that they fit in this GitHub diff view without scrolling, that would be great   :smiley_cat:  The GitHub diff view is ~120 characters wide, so I like to use that as a maximum width.

Rather than making two calls to `attr_accessible`, it would be a bit more idiomatic (IMO) to wrap the lines like this:
```ruby
  attr_accessible :firstname, :lastname, :status, :icsid, :department, :city, :state, :zipcode, :start_date,
                  :end_date , :title, :gender, :date_of_birth, :division1, :division2, :channels_attributes,
                  :title_ids, :comment
```